projects/ngx-maplibre-gl/src/lib/control/control.component.ts

```diff
@@ -57,28 +58,35 @@ export class CustomControl implements IControl {
   changeDetection: ChangeDetectionStrategy.OnPush,
   standalone: true,
 })
-export class ControlComponent<T extends IControl> implements OnDestroy {
+export class ControlComponent<T extends IControl> {
+  /** Init injection */
+  private readonly mapService = inject(MapService);
+  private readonly destroyRef = inject(DestroyRef);
+
   /** Init input */
-  @Input() position?: 'top-left' | 'top-right' | 'bottom-left' | 'bottom-right';
+  readonly position = input<
```

Comment:
Isn't there a type for this? Control position or something?

Reply:
Yes ControlPosition, i will change it !